SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -495,13 +493,13 @@ if ($PsBoundParameters.ContainsKey("shutdown") -or $PsBoundParameters.ContainsKe
                 Write-PowerManagementLogMessage -Type WARNING -Message "NSX Manager VMs have been stopped. NSX Edge VMs will not be handled automatically."
             } else {
                 Try {
-                    Write-output "adminpssword: $nsxMgrVIP.adminPassword"
+                    Write-Output "adminpssword: $nsxMgrVIP.adminPassword"
```

Comment:
`adminpsword`?

Reply:
This is left over from debugging - it is removed in the upcoming pull request.